code/main/task/eventProcessor.brs

```diff
@@ -22,66 +22,46 @@ function _adb_EventProcessor(task as object) as object
         _edgeModule: invalid,
         _consentModule: invalid,
         _mediaModule: invalid,
+        _modulesRegisteredForResponseEvents: invalid,
 
         init: function() as void
             m._configurationModule = _adb_ConfigurationModule()
             m._identityModule = _adb_IdentityModule(m._configurationModule)
             m._edgeModule = _adb_EdgeModule(m._configurationModule, m._identityModule)
             m._consentModule = _adb_ConsentModule(m._configurationModule)
-            m._mediaModule = _adb_MediaModule(m._configurationModule, m._edgeModule.createEdgeRequestQueue("MediaRequestQueue"))
-
+            m._mediaModule = _adb_MediaModule(m._configurationModule, m._edgeModule)
+            m._modulesRegisteredForResponseEvents = [m._consentModule, m._mediaModule]
             ' enable debug mode if needed
             if m._isInDebugMode()
                 _adb_serviceProvider().networkService._debugMode = true
             end if
         end function,
 
         handleEvent: function(event as dynamic) as void
-
             if _adb_isRequestEvent(event)
-                _adb_logInfo("EventProcessor::handleEvent() - Received request event:(" + chr(10) + FormatJson(event) + chr(10) + ")")
+                _adb_logVerbose("EventProcessor::handleEvent() - Received request event:(" + chr(10) + FormatJson(event) + chr(10) + ")")
 
                 try
                     m._processAdbRequestEvent(event)
                 catch ex
                     _adb_logError("EventProcessor::handleEvent() - Failed to process the request event, the exception message: " + ex.Message)
                 end try
-
-                if m._isInDebugMode()
-                    m._dumpDebugInfo(event, _adb_serviceProvider().loggingService, _adb_serviceProvider().networkService)
-                end if
-
             else
                 _adb_logWarning("EventProcessor::handleEvent() - Cannot handle event, invalid event: (" + chr(10) + FormatJson(event) + chr(10) + ")")
             end if
-        end function,
 
-        _isInDebugMode: function() as boolean
-            if m._task <> invalid and m._task.hasField("debugInfo")
-                return true
+            if m._isInDebugMode()
```

Comment:
I'd recommend reverting this change. If the task node accidentally receives non-Adobe data, this logic should not be executed.